python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1401,6 +1401,68 @@ async def is_m1m3_in_engineering_mode(self) -> bool:
 
         return m1m3_detailed_state in self.m1m3_engineering_states
 
+    async def run_m2_actuator_bump_test(
+        self,
+        actuator: int,
+        period: float,
```

Comment:
Consider adding a default value for the period.

Reply:
Set a default time of 60 seconds